app/assets/stylesheets/arclight/modules/hierarchy.scss

```diff
@@ -36,6 +36,13 @@
     line-height: 1.25;
   }
 
+  .al-document-abstract-or-scope-heading {
```

Comment:
What would you think about making this class a bit more generic?  Perhaps something that describes what it is, but is also generic enough that it could be used in other contexts.